src/setupTests.js

```diff
@@ -1,4 +1,6 @@
-global.beforeEach(() => {
+import { beforeEach } from "vitest";
```

Comment:
Rather than forcing all modules to be globally available during tests, I'm opting to deliberately import them. Feels cleaner.